src/lib/components/BucketAggregation/BucketAggregationValues.js

```diff
@@ -123,7 +123,7 @@ const ValueElement = (props) => {
   } = props;
   const label = bucket.label
     ? bucket.label
-    : `${keyField} (${bucket.doc_count})`;
+    : `${keyField} (${bucket.doc_count.toLocaleString('en-US')})`;
```

Comment:
I guess is fine for now to stick to `en-Us` locale but we should definitely integrate proper internationalization for this!